archive/frames/pagination.py

```diff
@@ -1,6 +1,6 @@
 from rest_framework.pagination import LimitOffsetPagination
+from archive.settings import PAGINATION_DEFAULT_LIMIT
 
 
 class LimitedLimitOffsetPagination(LimitOffsetPagination):
-    default_limit = 100
-    max_limit = 1000
```

Comment:
Is the max_limit still set somewhere, or should that be configurable?

Reply:
According to the [docs](https://www.django-rest-framework.org/api-guide/pagination/), the `max_limit` is only valid if the `page_query_param` is defined (which does not appear to be the case here).

Reply:
Hm, `page_query_param` seems to be a configurable attribute of the `PageNumberPagination` class. I'm not seeing anything in the docs about `page_query_param` for the `LimitOffsetPagination` class, but am I missing something?
https://www.django-rest-framework.org/api-guide/pagination/#limitoffsetpagination

Reply:
Oh I mixed them up! I'll add `max_limit` back.